ghcide/src/Development/IDE/Plugin/CodeAction.hs

```diff
@@ -100,7 +101,8 @@ codeAction lsp state (TextDocumentIdentifier uri) _range CodeActionContext{_diag
         [ CACodeAction $ CodeAction title (Just CodeActionQuickFix) (Just $ List [x]) (Just edit) Nothing
         | x <- xs, (title, tedit) <- suggestAction exportsMap ideOptions parsedModule text x
         , let edit = WorkspaceEdit (Just $ Map.singleton uri $ List tedit) Nothing
-        ] <> caRemoveRedundantImports parsedModule text diag xs uri
+        ] <> caRemoveRedundantExports parsedModule text diag xs uri
```

Comment:
Would you mind if I request to use a different name?
I mean, those undefined exports are not something "redundant", and it's more like, just as your title, "undefined", right?

Reply:
Indeed the name should change. It covers a bit more cases than `undefined`, like dodgy-exports. Maybe wrongExports or faultyExports?

Reply:
Yeah, they would be better.